next-app/src/theme.ts

```diff
@@ -0,0 +1,34 @@
+import { extendTheme, ThemeConfig } from "@chakra-ui/react";
+
+export const theme = extendTheme({
+  config: {
+    initialColorMode: "dark",
+    useSystemColorMode: false,
+  },
+  styles: {
+    global: () => ({
+      body: {
+        bg: "brand.900",
+        color: "white",
+      },
+    }),
+  },
+  colors: {
+    brand: {
+      900: "#200050", // Background color
+      800: "#371586", // Card background color
+    },
+  },
+  components: {
+    Card: {
+      baseStyle: {
+        borderRadius: "24px", // Card border radius
+      },
+    },
```

Comment:
Ovo si rekao da ne radi, ako imaš vremena i volje pogledaj ono što smo probavali na konzultacijama:
https://v2.chakra-ui.com/docs/components/card/theming

I istraži malo multipart komponente u chakri (tj. kako funkcioniraju) -> isto ako si voljan :D